### DISCUSSION TITLE
Brainstorming: Adição de novas páginas e elementos ao site.

### DISCUSSION THREAD

**Discussion by RodrigoEC** [OpenDevUFCG/opendevufcg.org#163] [Ideias]
**Descreva a feature em detalhes:**
Durante a reunião do core team sobre o site do portal no dia 09/03/2021 foi decidido que serão adicionados novos conteúdos ao site, transformando-o em um hub da OpenDev. Essa adição tem como objetivo centralizar todos os conteúdos criados e disponibilizados pela OpenDevUFCG e, dessa forma, facilitar a transição de um usuário entre os diversos projetos que a organização possui. Todavia, ainda não foi acordado exatamente o que deve ser adicionado ao site para que esse objetivo seja atingido, e é para isso que essa issue existe.

A issue tem como objetivo discutir esses novos elementos que serão adicionados ao site, como páginas ou seções novas, para que em reuniões futuras possamos definir propriamente quais os elementos que serão adicionados.

**Necessidade da feature:**
Para que o site se torne um hub da organização, são necessários novos elementos, como páginas de eventos, blog e projetos. Para concretizar esses elementos são necessárias novas ideias de elementos que podem ser adicionados ao site e também especificar mais detalhadamento os elementos citados previamente. 

**Como ela pode ser implementada:**
Deixe nos comentários sua ideia de algo que poderia ser adicionado ao portal, detalhando e explicando a ideia e o motivo da sua sugestão.

**Comment by JoseRenan** [comment]
Adicionar uma seção de blogs que lista os posts feitos no dev.to, acredito que existem formas nativas de fazer isso pelo dev.to mesmo, assim que postar lá, é incrementada na lista no site, mas ao clicar é redirecionado pro dev.to onde está o post

**Comment by RodrigoEC** [comment]
Adicionar uma página de eventos. Com o andromedev, o hacktoberfest e outros eventos vindo aí eu acho que é importante que o site do portal tenha uma página de eventos que sejam realmente um portal para esses outros sites, como o do hacktoberfest ou do andromedev.

Podiamos fazer uma estrutura de cards com cada evento.

**Comment by RodrigoEC** [comment]
Adicionar uma página de projetos. Acredito que pela quantidade de projetos que a OpenDev tem hoje em dia, uma página destinada a expor esses projetos seria muito interessante.

Existem projetos que possuem site e github, sendo assim precisamos pensar em uma forma legal de expor os dois, talvez fazer uma página para cada projeto (Ou um pop-up, não lembro o nome certo), aí dentro dessa estrutura mostrar as informações relevantes do projeto.

**Comment by RodrigoEC** [comment]
O site do Elas possui um "mural de avisos" para expor palestras, eventos e tudo mais que o elas propõe. Acho que seria legal a gente fazer algo parecido para a opendev, para os minicursos, palestras e eventos futuros

**Reply by JoseRenan** [reply]
Tu sabe dizer se no elas é feito usando um CMS da vida ou é na mão msm? adicionando o HTML sempre que tem coisa nova?

**Reply by RodrigoEC** [reply]
Rapaz, acho que @LeandraOliveiraS pode falar melhor sobre isso, ja que ela mexeu diretamente nisso

**Reply by LeandraOS** [reply]
A gente adiciona em um Enum de eventos. 

```
const events = [
    {
        title: "Café com Elas",
        description: '"Café com Elas" na programação da Jornada Acadêmica Computação@UFCG.',
        day: "5",
        month: "11",
        year: "2020",
        time: "20h",
        local: "Discord CCC",
        pageURL: "",
        eventURL: "",
    },
]

export default events;
```

Pra quem tem curiosidade de ver ainda mais, [nesse link](https://elas.computacao.ufcg.edu.br/DocsSiteElas/docs/site-enums) tem a documentação do site.

**Reply by LeandraOS** [reply]
Acho essa ideia ótima, porque fica tudo centralizado em um lugar só. No Elas tá bem assim, tá tendo o evento e tá tudo lá, quem tem interesse é só entrar, olhar e já vai tá tudo lá listado sem dificuldades pra encontrar a programação.

**Comment by RodrigoEC** [comment]
Adicionar botão de contribuição (algo que já tem no design que Huandrey fez)

**Reply by JoseRenan** [reply]
Acho que o botão de contribuição poderia ter uma página mostrando várias formas de contribuir, tipo, falando pra abrir o repositório, ler readme e contributing e tal, falar tbm se tiver algum tipo de issue que vc ganhe brinde e como fazer pra ganhar e tbm a questão de doação

**Reply by RodrigoEC** [reply]
Hmhumm, acho q cada projeto podia ter uma área de "como contribuir".

**Reply by JoseRenan** [reply]
Podia ser uma só tbm já que cada projeto tem seu contributing explicando direitinho :thinking: tipo, se cada um tem o seu, daí fica ruim de manter, atualizar em 2 lugares, nusei

**Reply by RodrigoEC** [reply]
Realmente..

**Comment by LeandraOS** [comment]
Acredito que uma boa ideia seria ter uma sessão de portfólio das atividades, eventos e etc do que a OpenDev já fez, imagens mostrando os eventos e tudo mais, para as pessoas conhecerem a história do que já foi feito e se interessar pelo o que também vem aí.

**Comment by LeandraOS** [comment]
Essa sugestão é mais pra o fim do desenvolvimento, mas seria muito interessante que a gente pudesse escrever uma documentação do portal. Porque assim, iria facilitar pra quem vai mexer futuramente e além disso pode servir até como material de estudo para o pessoal que tem interesse em aprender web. 

Uma docs do docusaurus seria legal. Porque iria cumprir o papel que eu citei acima e ainda quem fosse fazer ia aprender a mexer com o docusaurus.
